packages/components/src/ui/utils/get-valid-children.ts

```diff
@@ -12,11 +12,13 @@ import { Children, isValidElement } from '@wordpress/element';
 /**
  * Gets a collection of available children elements from a React component's children prop.
  *
- * @param {import('react').ReactNode} children
+ * @param children
  *
- * @return {import('react').ReactNodeArray} An array of available children.
+ * @return An array of available children.
  */
-export function getValidChildren( children: ReactNode ): ReactNodeArray {
+export function getValidChildren(
+	children: ReactNode
+): ( ReactElement | ReactNode )[] {
```

Comment:
Is there a particular reason for using `( ReactElement | ReactNode )[]` instead of `ReactNodeArray` ?

Reply:
Oh huh, they should be equivalent 🤔